iree/vm/BUILD

```diff
@@ -195,24 +194,24 @@ cc_library(
     ],
 )
 
-# TODO(benvanik): see if we can remove this; not good to have this dep.
-gentbl(
-    name = "bytecode_op_table_gen",
-    tbl_outs = [
-        ("-gen-iree-vm-op-table-defs", "bytecode_op_table.h"),
-    ],
-    tblgen = "//iree/tools:iree-tblgen",
-    td_file = "//iree/compiler/Dialect/VM/IR:VMOps.td",
-    td_srcs = [
-        "//iree/compiler/Dialect/IREE/IR:td_files",
-        "//iree/compiler/Dialect/VM/IR:td_files",
-        "@llvm-project//mlir:OpBaseTdFiles",
-        "@llvm-project//mlir:include/mlir/IR/SymbolInterfaces.td",
-        "@llvm-project//mlir:include/mlir/Interfaces/CallInterfaces.td",
-        "@llvm-project//mlir:include/mlir/Interfaces/ControlFlowInterfaces.td",
-        "@llvm-project//mlir:SideEffectTdFiles",
-    ],
-)
+# TODO(#357): Add a script to update bytecode_op_table.h.
+# gentbl(
+#     name = "bytecode_op_table_gen",
+#     tbl_outs = [
+#         ("-gen-iree-vm-op-table-defs", "bytecode_op_table.h"),
+#     ],
+#     tblgen = "//third_party/iree/iree/tools:iree-tblgen",
+#     td_file = "//third_party/iree/iree/compiler/Dialect/VM/IR:VMOps.td",
+#     td_srcs = [
+#         "//third_party/iree/iree/compiler/Dialect/IREE/IR:td_files",
+#         "//third_party/iree/iree/compiler/Dialect/VM/IR:td_files",
+#         "//third_party/llvm/llvm-project/mlir:OpBaseTdFiles",
+#         "//third_party/llvm/llvm-project/mlir:include/mlir/IR/SymbolInterfaces.td",
+#         "//third_party/llvm/llvm-project/mlir:include/mlir/Interfaces/CallInterfaces.td",
+#         "//third_party/llvm/llvm-project/mlir:include/mlir/Interfaces/ControlFlowInterfaces.td",
+#         "//third_party/llvm/llvm-project/mlir:SideEffectTdFiles",
```

Comment:
keep open-source style paths in these :P 

![image](https://user-images.githubusercontent.com/4010439/101212008-e26a2480-362c-11eb-8e2d-595488735466.png)

Reply:
Whoops, thank you!